### ISSUE TITLE
Add a README file to this project

### ISSUE THREAD

**Issue by SarangShah87** [distilledpm/archives#4]
I'm considering adding a README to the project with:

- an explanation of Distilled archives repository
- instructions on how to build
- instructions on how to make edits and submit PRs (???)

**Comment by savelbr** [comment]
Good idea. The first two points should definitely go in a README. The last point is often moved into a separate CONTRIBUTING file, but I've seen it in the README as well. All depends on how long it is, I guess.

**Comment by savelbr** [comment]
On second thought, seen our target contributors might not necessarily be familiar with Github, it's probably best to keep the last point also in the README, where it's more visible.
